probe-rs/src/coresight/ap_access.rs

```diff
@@ -15,7 +15,26 @@ where
     type Error: std::error::Error;
     fn read_register_ap(&mut self, port: PORT, register: REGISTER)
         -> Result<REGISTER, Self::Error>;
+
+    /// Read a register using a block transfer. This can be used
+    /// to read multiple values to the same register.
+    fn read_block_ap(
```

Comment:
I think we should call this read_register_ap_block or block_read_register_ap (same goes for write) to be consistent with the existing naming scheme and avoid confusion to what is being written.

Reply:
Renamed to `read_ap_register_repeated`.